api/src/routers/meals.js

```diff
@@ -0,0 +1,87 @@
+import express from "express";
+import { StatusCodes } from "http-status-codes/build/cjs/index.js";
```

Comment:
You can direct import after install .
https://www.npmjs.com/package/http-status-codes